src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -35,11 +35,11 @@ class RatedDisabilityList extends React.Component {
           <p>
             If you get this error again, please call the VA.gov help desk at{' '}
             <a
-              href="tel:18555747286"
-              aria-label="1. 8 5 5. 5 7 4. 7 2 8 6."
-              title="Dial the telephone number 1-855-574-7286"
+              href="tel:8446982311"
+              aria-label="8 4 4. 6 9 8. 2 3 1 1."
```

Comment:
Ooh, interesting. Out of curiosity, @1Copenut is this best practices?

Reply:
@cvalarida It's a new practice we're trying out, from this article by That Dev Girl: https://thatdevgirl.com/blog/accessibility-phone-number-formatting

It makes phone numbers a lot easier to parse when they're read back by AT.